template/README.md.jinja

```diff
@@ -22,7 +22,7 @@
 
 ## How to use {{ package_name }}
 
-{{ package_short_description }}
+{{ package_short_description|replace('\"', '\\\"') }}
```

Comment:
Is it necessary to escape double quotes in a README file? At least for GitHub formatting it seems to be fine both with (\") and without (") escaping

Reply:
True, might not needed. But this is the original feature from the cookiecutter that we had.